functests/utils/pods/pods.go

```diff
@@ -223,3 +224,27 @@ func GetPerformanceOperatorPod() (*corev1.Pod, error) {
 
 	return &pods.Items[0], nil
 }
+
+//GetCpusNumber returns the cpus number that are used by mco pod
```

Comment:
by MCO pod or by the pod passed as argument?

Reply:
yes by the object pod, will modify that, Thank you for noticing me!